### PR TITLE
NATIVE-454: Upgrade react-native to v0.61.5

### PR DESCRIPTION
**Pull request by klinzo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
     "generator-rn-toolbox": "^3.11.1",
     "get-dev-paths": "^0.1.1",
     "http-proxy": "^1.17.0",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
```
Comment (maxammann) [inline package.json]: The last time I checked there were no jest types for v25 yet. I think it is best to stick with 24 until they are available.

Comment (maxammann) [reply package.json]: Or is this recommended by the diff? In this case it is still weird if there are no flow types.

Comment (klinzo) [reply package.json]: Yep, good catch :+1:

File: src/modules/i18n/containers/__tests__/I18nProviderContainer.spec.js [hunk 2]
```diff
@@ -13,78 +13,84 @@ jest.mock('../../../i18n/LanguageDetector')
 
 const mockStore = configureMockStore()
 
+// jest.resetModules() produces a invariant violation: Invalid hook call https://github.com/facebook/jest/issues/8987
```
Comment (Schedulaar) [inline src/modules/i18n/containers/__tests__/I18nProviderContainer.spec.js]: Is this also required although you downgraded jest again?

**Comment by Schedulaar** [comment]
For me `yarn android:release` doesn't work. It quits with the message:
```
> Configure project :react-native-webview
:react-native-webview:reactNativeAndroidRoot /home/michael/StudioProjects/integreat-react-native-app/node_modules/react-native/android

> Task :app:bundleReleaseJsAndAssets FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5/userguide/command_line_interface.html#sec:command_line_warnings
71 actionable tasks: 2 executed, 69 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:bundleReleaseJsAndAssets'.
> Could not list contents of '/home/michael/StudioProjects/integreat-react-native-app/node_modules/istanbul-lib-instrument/node_modules/@babel/generator/node_modules/.bin/jsesc'. Couldn't follow symbolic link.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s

error Failed to install the app. Make sure you have the Android development environment set up: https://facebook.github.io/react-native/docs/getting-started.html#android-development-environment. Run CLI with --verbose flag for more details.
Error: Command failed: ./gradlew app:installRelease -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:bundleReleaseJsAndAssets'.
> Could not list contents of '/home/michael/StudioProjects/integreat-react-native-app/node_modules/istanbul-lib-instrument/node_modules/@babel/generator/node_modules/.bin/jsesc'. Couldn't follow symbolic link.

```

File: src/modules/app/__tests__/createNavigateToCategory.spec.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ describe('createNavigateToCategory', () => {
     expect(navigation.navigate).toHaveBeenCalledWith(expect.objectContaining({
       key: expect.stringMatching(/^.{6,}$/) // at least 6 chars but no newline
     }))
+    // $FlowFixMe .mock is missing
```
[the remark this answers is not in the document]
Comment (klinzo) [reply src/modules/app/__tests__/createNavigateToCategory.spec.js]: Yes, also required after the downgrade. Changed for some occurances

File: src/modules/i18n/containers/__tests__/I18nProviderContainer.spec.js (hunk 2 above)
Comment (klinzo) [reply src/modules/i18n/containers/__tests__/I18nProviderContainer.spec.js]: Yes, sadly still required

**Comment by klinzo** [comment]
@Schedulaar `yarn android:release` works for me. Could you try removing `node_modules` and executing it again?

**Comment by Schedulaar** [comment]
Yeah, works for me too, now.
I had to clean the builds for some reason :thinking:

**Pull request merged**